src/Common/Infrastructure/Controllers/Attributes/ApiKeyAuthAttribute.cs

```diff
@@ -27,7 +27,7 @@ public ApiKeyAuthAttribute(string requiredAccess)
         public Task OnActionExecutionApiKeyAsync(ActionExecutingContext context, ActionExecutionDelegate next)
         {
             if (!context.HttpContext.Request.Headers.TryGetValue(ApiKeyHeaderName, out var potentialHeaderApiKey)
-                && !context.HttpContext.Request.Query.TryGetValue(ApiKeyQueryName, out var potentialQueryApiKey))
+                & !context.HttpContext.Request.Query.TryGetValue(ApiKeyQueryName, out var potentialQueryApiKey))
```

Comment:
You dropped an &?

Reply:
Compiler rightfully errors (since net6?) that the variable `potentialQueryApiKey` is otherwise undeclared in the if check below. By using this construct we now are sure both are declared.